inception/inception-app-webapp/src/test/java/de/tudarmstadt/ukp/inception/db/InceptionHsqldbIntegrationTest.java

```diff
@@ -55,6 +58,8 @@ static void tearDownClass()
     }
 
     @Nested
+    @ContextConfiguration
```

Comment:
(Why) did you have to add these annotations here?

Reply:
Probably a side effect of a refactor, I didn't changed that class - will revert that